.hlint.yaml

```diff
@@ -11,6 +11,7 @@
 - ignore: { name: Avoid lambda using `infix` }
 - ignore: { name: Eta reduce }
 - ignore: { name: Use section }
+- ignore: { name: "Use :" }
```

Comment:
this makes you write pointfree stuff that makes things hard to read; I think it will be uncontroversial that we shouldn't sacrifice readability to use trickshots, e.g. 
`... (: [])` vs `... \device -> [device]`

Reply:
ah, but of course turning off the rule is fine with me, if somebody doesn't like it.

Reply:
alright I will revert that then :3